cvat-ui/src/cvat-core-wrapper.ts

```diff
@@ -5,8 +5,10 @@
 import _cvat from 'cvat-core/src/api';
 
 const cvat: any = _cvat;
+const prefix: string = '/cvat-annotation';
```

Comment:
I'm confused about why this config is set here rather than in config.js...

Reply:
The cvat-ui doesn't have a config.js file for now. It only exists in cvat-core dir. ofc we can create one.